lib/cli/src/automigrate/index.ts

```diff
@@ -61,4 +64,8 @@ export const automigrate = async ({ fixId, dryRun, yes }: FixOptions = {}) => {
       }
     }
   }
+
+  logger.info();
+  logger.info('✅ migration check successfully ran');
```

Comment:
Feel free to suggest better texts for these.

Looks like this (this was before I added extra logs for spaces around)
![image](https://user-images.githubusercontent.com/1671563/169872294-165114f4-e2a4-486c-8291-2e77951784ad.png)